res/values/setup.xml

```diff
@@ -30,7 +30,7 @@
     <!-- Colors -->
     <color name="login_background_color">#FFFFFF</color>
     <color name="login_logo_background_color">#FFFFFF</color>
-    <color name="background_color">#F7F7F7</color>
+    <color name="background_color">#FFFFFF</color>
```

Comment:
I didn't see any either but changed it to white, since @jancborchardt opened a ticket to use white as the background color value.